### ISSUE TITLE
[COVER ART] Worms Rumble

### ISSUE THREAD

**Issue by Kirbo** [ktnrg45/pyps4-homeassistant#50]
**Media Data**
```
  "CUSA23465": {
    "media_title": "Worms Rumble",
    "media_image_url": null,
    "media_content_type": null,
    "locked": false
  }
```

**Title of Media**: "Worms Rumble"

**SKU ID**: "CUSA23465"

**Region**: "Europe"

**Comment by Kirbo** [comment]
```
2021-01-06 16:29:29 ERROR (MainThread) [homeassistant.components.ps4.media_player] Could not find data in region: Finland for PS ID: CUSA23465
```

**Comment by ktnrg45** [comment]
This has been fixed in release 1.2.0.
